src/doc/src/reference/unstable.md

```diff
@@ -255,3 +255,23 @@ etc.).
 This also changes the way Cargo interacts with the compiler, helping to
 prevent interleaved messages when multiple crates attempt to display a message
 at the same time.
+
+### timings
+
+The `timings` feature gives some information about how long each compilation
+takes, and tracks concurrency information over time.
+
+```
+cargo +nightly build -Z timings
+```
+
+The `-Ztimings` flag can optionally take a comma-separated list of the
+following values:
+
+- `html` — Saves a file called `cargo-timing.html` to the current directory
+  with a report of the compilation.
+- `info` — Displays a message to stdout after each compilation finishes with
+  how long it took.
```

Comment:
This is mostly doing what rustbuild does (with RUSTC-TIMING).  Not sure if it can actually replace it, but if it was valuable there it might be useful for others?

Reply:
Eh I think it's fine for now, it's always something we can iterate on